language-design.md

```diff
@@ -106,7 +172,17 @@ you do so? If not, why not?
 
 **Response**
 
+From the selected quotes, the two experiences of natural language seem to be at odds with one another. The Pavlus quote claims that users preferred working with something akin to a natural language, whereas the Cook quote says that programmers found natural language to be too ambiguous. These two experiences cannot be directly compared without examining the backgrounds of the people being quoted as well as the amount of exposure that they had to the language. 
 
```

Comment:
I think background is an important point, especially in the case of non-native English speakers. Given that natural language tends to mean "everyday English" (at least in the examples given), this could make a big difference in how accessible a language is.